todo_app/app.py

```diff
@@ -1,11 +1,23 @@
-from flask import Flask
-
+from flask import Flask, render_template, request, redirect, url_for
+from todo_app.data.session_items import add_item, get_items
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
-
 @app.route('/')
 def index():
-    return 'Hello World!'
+    return render_template('index.html', items=get_items(), route='display')
+
+@app.route('/add', methods = ["GET","POST"])
+def add():
+    if (request.method== "GET" ):
+        return render_template('add.html')
```

Comment:
It looks like `add.html` hasn't been created, so should we get rid of the GET part of this method? 😄